knowledge-hub/labels.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Label Management"
+description: "Creating, managing and tuning labels for feedback triage."
+---
+
+## Overview
+
+Project area labels and tag labels are workspace-defined labels that help triage feedback snippets.
+When auto-triage is enabled for a particular label, Monterey will go ahead and auto assign a project area label when the feedback is ingested or the labels are backfilled, provided our AI models are confident in that prediction.
+
+While project area labels and tag labels work functionally similar in downstream uses (filtering, chat, reports, etc.), they have one key difference that necessitates the separation of the two.
+
+For a particular snippet when auto-triage is enabled:
+
+- 1 or 0 project areas can be auto-assigned
+- Many or 0 tag areas can be auto-assigned
```

Comment:
nit: and not blocking, but my mind feels like this is backwards and should be `0 or 1` or `0 to many`